DataMigration/src/test/java/org/sagebionetworks/dian/datamigration/PathsHelperTests.java

```diff
@@ -160,12 +161,17 @@ public void test_findAllJsonFilesInFolder() throws IOException {
         assertNotNull(files);
         assertEquals(6, files.size());
         // Sort so that we can access by index
-        files.sort((o1, o2) -> o1.getFileName().toString()
-                .compareTo(o2.getFileName().toString()));
+        files.sort((o1, o2) -> {
+            if (o1.getFileName().toString().equals(o2.getFileName().toString())) {
```

Comment:
@brucehoff Updated to do a second level sort in the case the filenames are the same, that way I can predictably determine the files later in the assertions.